src/filemap.rs

```diff
@@ -80,7 +80,7 @@ pub fn write_system_newlines<T>(writer: T,
     }
 }
 
-fn write_file<T>(text: &StringBuffer,
+pub fn write_file<T>(text: &StringBuffer,
                  filename: &str,
```

Comment:
please adjust the indentation on following lines (surprised this doesn't fail tests)